src/neuroconv/tools/testing/mock_interfaces.py

```diff
@@ -162,24 +165,52 @@ class MockImagingInterface(BaseImagingExtractorInterface):
     A mock imaging interface for testing purposes.
     """
 
+    ExtractorModuleName = "roiextractors.testing"
```

Comment:
What do these changes have to do with a MockSegmentationInterface?

Reply:
This is related to make the MockImagingInterface to work with some of the magic in the tests mixings. I can make this changes in a different PR if that is what you are asking.

Reply:
Yeah, I think this is a good idea anyways. Check:

#1076